Error: no prebuilt wasm-opt binaries are available for this platform: ARM (Mac M2 processor)

### What happened?

I am running a Dev Container on my M2 Mac mini, and got this error when I run `task run-web` This a continuation of #2402 - the initial problem was solved, but I now encounter this problem. In the original ticket, @max-sixty wrote "Thanks for the report. I think this is https://github.com/PRQL/prql/pull/2420. We also will need to merge https://github.com/PRQL/prql/pull/2421."

There's no particular rush needed: I can work fine on my Intel Mac, but I didn't want this report to get lost. Thanks

```
vscode ➜ /workspaces/prql (cache-npm) $ task run-web
task: [build-web] cd website && hugo --minify
Start building sites … 
hugo v0.111.3-5d4eb5154e1fed125ca8e9b5a0315c4180dab192 linux/arm64 BuildDate=2023-03-12T11:40:50Z VendorInfo=gohugoio

                   | EN  
-------------------+-----
  Pages            | 16  
  Paginator pages  |  0  
  Non-page files   |  6  
  Static files     | 25  
  Processed images |  0  
  Aliases          |  0  
  Sitemaps         |  1  
  Cleaned          |  0  

Total in 163 ms
task: [build-web] cd book && mdbook build
2023-05-25 02:31:45 [INFO] (mdbook::book): Book building has started
Warning: The mdbook-admonish preprocessor was built against version 0.4.21 of mdbook, but we're being called from version 0.4.28
    Finished dev [unoptimized + debuginfo] target(s) in 0.28s
     Running `/workspaces/prql/target/debug/mdbook-prql supports html`
Running comparison preprocessor
    Finished dev [unoptimized + debuginfo] target(s) in 0.24s
     Running `/workspaces/prql/target/debug/mdbook-prql`
Running comparison preprocessor
Running comparison preprocessor
2023-05-25 02:31:50 [INFO] (mdbook::book): Running the html backend
task: [build-web] mkdir -p website/public
task: [build-web] rsync -ai --checksum --delete book/book/ website/public/book/
.d..t...... ./
.f..t...... .nojekyll
>fc.t...... 404.html
.f..t...... ayu-highlight.css
.f..t...... book.js
>fcst...... changelog.html
.f..t...... clipboard.min.js
.f..t...... comparison-table.css
.f..t...... elasticlunr.min.js
.f..t...... favicon.png
.f..t...... favicon.svg
.f..t...... highlight-prql.js
.f..t...... highlight.css
.f..t...... highlight.js
>fc.t...... index.html
>fc.t...... introduction.html
>fcst...... lib.rs
.f..t...... main.rs
.f..t...... mark.min.js
.f..t...... mdbook-admonish.css
>fcst...... print.html
.f..t...... searcher.js
>fcst...... searchindex.js
>fcst...... searchindex.json
.f..t...... tomorrow-night.css
*deleting   integrations/cli.html
.d..t...... FontAwesome/
.d..t...... FontAwesome/css/
.f..t...... FontAwesome/css/font-awesome.css
.d..t...... FontAwesome/fonts/
.f..t...... FontAwesome/fonts/FontAwesome.ttf
.f..t...... FontAwesome/fonts/fontawesome-webfont.eot
.f..t...... FontAwesome/fonts/fontawesome-webfont.svg
.f..t...... FontAwesome/fonts/fontawesome-webfont.ttf
.f..t...... FontAwesome/fonts/fontawesome-webfont.woff
.f..t...... FontAwesome/fonts/fontawesome-webfont.woff2
.d..t...... bindings/
>fc.t...... bindings/dotnet.html
>fc.t...... bindings/elixir.html
>fc.t...... bindings/index.html
>fc.t...... bindings/java.html
>fc.t...... bindings/javascript.html
>fc.t...... bindings/python.html
>fc.t...... bindings/r.html
>fc.t...... bindings/rust.html
.d..t...... contributing/
>fc.t...... contributing/developing-with-dev-containers.html
>fc.t...... contributing/developing-with-docker.html
>fc.t...... contributing/development.html
.f..t...... contributing/fortnightly-dev-call.ics
>fcst...... contributing/index.html
.d..t...... css/
.f..t...... css/chrome.css
.f..t...... css/general.css
.f..t...... css/print.css
.f..t...... css/variables.css
.d..t...... examples/
>fcst...... examples/cte.html
>fc.t...... examples/employees.html
>fc.t...... examples/index.html
>fc.t...... examples/list-equivalence.html
>fc.t...... examples/variables.html
.d..t...... fonts/
.f..t...... fonts/OPEN-SANS-LICENSE.txt
.f..t...... fonts/SOURCE-CODE-PRO-LICENSE.txt
.f..t...... fonts/fonts.css
.f..t...... fonts/open-sans-v17-all-charsets-300.woff2
.f..t...... fonts/open-sans-v17-all-charsets-300italic.woff2
.f..t...... fonts/open-sans-v17-all-charsets-600.woff2
.f..t...... fonts/open-sans-v17-all-charsets-600italic.woff2
.f..t...... fonts/open-sans-v17-all-charsets-700.woff2
.f..t...... fonts/open-sans-v17-all-charsets-700italic.woff2
.f..t...... fonts/open-sans-v17-all-charsets-800.woff2
.f..t...... fonts/open-sans-v17-all-charsets-800italic.woff2
.f..t...... fonts/open-sans-v17-all-charsets-italic.woff2
.f..t...... fonts/open-sans-v17-all-charsets-regular.woff2
.f..t...... fonts/source-code-pro-v11-all-charsets-500.woff2
.d..t...... integrations/
>f+++++++++ integrations/dbt.html
>fc.t...... integrations/duckdb.html
>fc.t...... integrations/index.html
>fc.t...... integrations/jupyter.html
>fc.t...... integrations/prefect.html
>fc.t...... integrations/rill.html
>fc.t...... integrations/vscode.html
.d..t...... internals/
>fc.t...... internals/compiler-architecture.html
>fc.t...... internals/functional-lang.html
>fc.t...... internals/helpers.html
>fc.t...... internals/index.html
>fc.t...... internals/name-resolving.html
>fc.t...... internals/syntax-highlighting.html
.d..t...... language-features/
>fc.t...... language-features/case.html
>fc.t...... language-features/coalesce.html
>fc.t...... language-features/dates-and-times.html
>fc.t...... language-features/distinct.html
>fc.t...... language-features/f-strings.html
>fc.t...... language-features/index.html
>fc.t...... language-features/null.html
>fc.t...... language-features/ranges.html
>fcst...... language-features/regex.html
>fcst...... language-features/s-strings.html
>fc.t...... language-features/strings.html
>fc.t...... language-features/target.html
.d..t...... language-features/standard-library/
>fc.t...... language-features/standard-library/from_text.html
>fcst...... language-features/standard-library/index.html
>fc.t...... language-features/standard-library/loop.html
>fc.t...... language-features/standard-library/reading-files.html
.d..t...... queries/
>fcst...... queries/functions.html
>fc.t...... queries/index.html
>fcst...... queries/pipelines.html
>fcst...... queries/variables.html
.d..t...... syntax/
>fc.t...... syntax/comments.html
>fcst...... syntax/expressions-and-operators.html
>fcst...... syntax/index.html
>fc.t...... syntax/inner-transforms.html
>fcst...... syntax/keywords.html
>fc.t...... syntax/lists.html
>fc.t...... syntax/numbers.html
>fc.t...... syntax/parameters.html
>fc.t...... syntax/pipes.html
>fcst...... syntax/quoted-identifiers.html
.d..t...... transforms/
>fc.t...... transforms/aggregate.html
>fcst...... transforms/append.html
>fc.t...... transforms/derive.html
>fc.t...... transforms/filter.html
>fc.t...... transforms/from.html
>fc.t...... transforms/group.html
>fc.t...... transforms/index.html
>fc.t...... transforms/join.html
>fcst...... transforms/select.html
>fcst...... transforms/sort.html
>fc.t...... transforms/take.html
>fc.t...... transforms/window.html
task: [install-npm-dependencies-if-needed] npm install --install-links=false && touch node_modules/.timestamp
npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
npm ERR! code 1
npm ERR! path /workspaces/prql/bindings/prql-js
npm ERR! command failed
npm ERR! command sh -c npm run build
npm ERR! > prql-js@0.8.1 build
npm ERR! > npm run build:node && npm run build:bundler && npm run build:web
npm ERR! 
npm ERR! 
npm ERR! > prql-js@0.8.1 build:node
npm ERR! > wasm-pack build --target nodejs --release --out-dir dist/node && rm dist/node/.gitignore
npm ERR! [INFO]: 🎯  Checking for the Wasm target...
npm ERR! [INFO]: 🌀  Compiling to Wasm...
npm ERR!    Compiling aho-corasick v1.0.1
npm ERR!    Compiling regex-syntax v0.7.1
npm ERR!    Compiling regex v1.8.1
npm ERR!    Compiling prql-compiler v0.8.1 (/workspaces/prql/prql-compiler)
npm ERR!    Compiling prql-js v0.8.1 (/workspaces/prql/bindings/prql-js)
npm ERR!     Finished release [optimized] target(s) in 29.62s
npm ERR! [INFO]: License key is set in Cargo.toml but no LICENSE file(s) were found; Please add the LICENSE file(s) to your project directory
npm ERR! [INFO]: ⬇️  Installing wasm-bindgen...
npm ERR! Error: no prebuilt wasm-opt binaries are available for this platform: Unrecognized target!
npm ERR! To disable `wasm-opt`, add `wasm-opt = false` to your package metadata in your `Cargo.toml`.
npm ERR! Caused by: no prebuilt wasm-opt binaries are available for this platform: Unrecognized target!
npm ERR! To disable `wasm-opt`, add `wasm-opt = false` to your package metadata in your `Cargo.toml`.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vscode/.npm/_logs/2023-05-25T02_31_51_998Z-debug-0.log
task: Failed to run task "run-web": task: Failed to run task "build-web": task: Failed to run task "install-npm-dependencies-if-needed": exit status 1
vscode ➜ /workspaces/prql (cache-npm) $  
```

### PRQL input

```elm
N/A
```


### SQL output

```SQL
N/A
```


### Expected SQL output

```SQL
N/A
```


### MVCE confirmation

- [X] Minimal example
- [X] New issue

### Anything else?

_No response_

--- a/Taskfile.yml
+++ b/Taskfile.yml
@@ -23,7 +23,8 @@ vars:
     rust-lang.rust-analyzer
   cargo_crates:
     bacon cargo-audit cargo-insta cargo-release default-target mdbook
-    mdbook-admonish mdbook-footnote mdbook-toc wasm-bindgen-cli wasm-pack
+    mdbook-admonish mdbook-footnote mdbook-toc wasm-bindgen-cli wasm-opt
+    wasm-pack
 
 tasks:
   # main installer is "setup-dev" which calls other tasks

